bindings/gumjs/gumcmodule.c

```diff
@@ -173,33 +231,37 @@ gum_cmodule_free (GumCModule * cmodule)
     gum_memory_free (GSIZE_TO_POINTER (r->base_address), r->size);
   }
```

Comment:
`::dispose()` needs to handle being run more than once. It should however be used for releasing references to other objects. Memory should be freed in `::finalize()`, and that can only run once. So here we should probably be using `::finalize()`.